packages/core/src/components/alert/alert.scss

```diff
@@ -0,0 +1,104 @@
+:host {
+  display: block;
+
+  --fl-alert-border-radius: 4px;
+  --fl-alert-position: absolute;
+
+  //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+  // Alert color tokens
+  //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+  // Success Colors
+  --fl-alert-success-fill: #eeffee;
+  --fl-alert-success-stroke: #27b43b;
+
+  // Warning Colors
+  --fl-alert-warning-fill: #fffbee;
+  --fl-alert-warning-stroke: #feb72c;
+
+  // Error Colors
+  --fl-alert-error-fill: #ffefee;
+  --fl-alert-error-stroke: #ed443d;
+
+  // Info Colors
+  --fl-alert-info-fill: #eef9ff;
+  --fl-alert-info-stroke: #33c2ff;
```

Comment:
If you pull from `main`, there should be CSS vars for `success`, `error`... You could use those values instead of re-defining them here.

So 
```
--fl-alert-error-fill: var(--fl-color-error-tint);
--fl-alert-error-stroke: var(--fl-color-error);
```

Reply:
Yeah saw those but the Figma used a different color, which one should I go with?

Reply:
I'd say let's be consistent and go with what's in the codebase